device/marvell/armhf-marvell_et6448m_52x-r0/led_control.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+#
+# led_control.py
+#
+# Platform-specific LED control functionality for SONiC
+#
+
+try:
+    from sonic_led.led_control_base import LedControlBase
+except ImportError, e:
+    raise ImportError(str(e) + " - required module not found")
+
+
+
+class LedControl(LedControlBase):
+    """Platform specific LED control class"""
+    def port_link_state_change(self, portname, state):
+        return
```

Comment:
If you are not using ledd to control the LEDs on this platform, you can use the new mechanism to disable the daemon altogether. See this PR https://github.com/Azure/sonic-buildimage/pull/3920 for an example.

Reply:
@jleveque comment addressed